src/createSelectable.js

```diff
@@ -16,11 +16,15 @@ const createSelectable = (WrappedComponent) => {
 
 
 		render () {
-			return React.createElement(
+			let ele = React.createElement(
 				WrappedComponent,
 				this.props,
 				this.props.children
 			);
+
+			return <div id={"selectableItem-"+this.props.selectableKey}>
+				{ele}
+			</div>
```

Comment:
Can you not replace the createElement call above here too, e.g.

```jsx
return (
  <div id={"selectableItem-"+this.props.selectableKey}>
    <WrappedComponent {..this.props}>
      {...this.props.children}
    </WrappedComponent>
  </div>
);
```

Reply:
Yes I can :D